client/internal/statemachine/readinstance_by_name.go

```diff
@@ -0,0 +1,46 @@
+package statemachine
+
+import (
+	"context"
+	"fmt"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/http"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/url"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/model/statemachine"
+)
+
+type ReadInstanceByNameInput struct {
+	Name string
+}
+
+func NewReadInstanceByNameInput(name string) ReadInstanceByNameInput {
+	return ReadInstanceByNameInput{
+		Name: name,
+	}
+}
+
+type ReadInstanceByNameOutput = statemachine.Instance
+
+var NewReadInstanceByNameOutputBuilder = statemachine.NewInstanceBuilder
+
+func ReadInstanceByName(ctx context.Context, client http.Client, readInp ReadInstanceByNameInput) (*ReadInstanceByNameOutput, error) {
```

Comment:
I don't see this code being used anywhere. Do we really need to add it into this PR?

Reply:
removed 👍